src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -1027,3 +1027,19 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    arg_type=get_enum_type(['none', 'minimal', 'full']),
                    help='Specifies how much metadata to include in the response payload.')
         c.argument('marker', validator=validate_marker, nargs='+')
+
+    with self.argument_context('storage fs create') as c:
+        c.extra('file_system_name', options_list=['--name', '-n'],
+                help="File system name.", required=True)
+
+    with self.argument_context('storage fs directory create') as c:
+        c.extra('file_system_name',
+                help="File system name.", required=True)
+        c.extra('directory_name', options_list=['--name', '-n'],
+                help="The name of directory.", required=True)
+
+    with self.argument_context('storage fs file create') as c:
+        c.extra('file_system_name',
+                help="File system name.", required=True)
+        c.extra('file_path',
+                help="The file path.", required=True)
```

Comment:
Will it be more convenient for users to input and remember the `file_system_name` parameter of several commands with the same option name?
For example: `file_system_name` ->`-s`, `directory_name` ->`-d`, `file_path` -> `-p`